meta-iot2000-bsp/recipes-support/swupdate/swupdate_%.bbappend

```diff
@@ -22,6 +22,8 @@ do_install_append() {
     install -d ${D}${sysconfdir}
     install -m 644 ${WORKDIR}/hwrevision ${D}${sysconfdir}
     install -m 644 ${WORKDIR}/swupdate.cfg ${D}${sysconfdir}
+
+    sed -i "s/hawkbit/${HAWKBIT_SERVER_IP}/g" ${D}${systemd_unitdir}/system/swupdate.service
```

Comment:
we may want to make the complete URL user-configurable

Reply:
update the service file to have a clear pattern. e.g. @hawkbit_url@ and use sed to replace it with the HAWKBIT_SERVER_URL setting from local.conf (or its default value, which needs to be defined somewhere)